src/__tests__/starWarsData.ts

```diff
@@ -6,7 +6,7 @@
  * JSON objects in a more complex demo.
  */
 
-interface Ship {
+export interface Ship {
```

Comment:
why do you need this export?

Reply:
I am as trying something and forgot to revert this